example/initialization/1_b_secret_function.html

```diff
@@ -10,6 +10,7 @@
       html[hidden] { display: none; }
     </style>
     <script src="http://localhost:8080/xfc.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.23.0/polyfill.min.js"></script>
```

Comment:
same here, the polyfill here is not used for xfc, but for a Promise call on this page.